cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/LanguageTraits.kt

```diff
@@ -227,3 +244,23 @@ inline infix fun <reified T : HasOverloadedOperation> KClass<T>.of(
 ): Pair<KClass<T>, String> {
     return Pair(T::class, operatorCode)
 }
+
+/** Checks whether the name for a function (as [CharSequence]) is a known operator name. */
+context(LanguageProvider)
+val CharSequence.isKnownOperatorName: Boolean
+    get() {
+        val language = language
+        if (language !is HasOperatorOverloading) {
+            return false
+        }
+
+        // If this is a parsed name, we only are interested in the local name
```

Comment:
Why are we only interested in the local name when it is a parsed name, what is the other case?

Reply:
In any other case its a regular String of CharSequence, without any "name" semantics, in this case we can just take the full name